src/Silex/ArgumentResolver.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Silex framework.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Silex;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Controller\ArgumentResolverInterface;
+
+/**
+ * HttpKernel Argument Resolver for Silex.
+ *
+ * @author Romain Neutron <imprec@gmail.com>
+ */
+class ArgumentResolver implements ArgumentResolverInterface
+{
+    private $ar;
+    private $app;
+
+    public function __construct(ArgumentResolverInterface $ar)
+    {
+        $this->ar = $ar;
```

Comment:
Short variable names are bad for readability. What about naming it `$argumentResolver`?